test/conftest.py

```diff
@@ -53,11 +56,22 @@ def insert_data_from_csv(in_csv_file, rec_count=-1):
     return insert_data_from_csv
 
 
-@pytest.fixture()
+@pytest.fixture(scope="class")
 def utils(db):
     return Utils(db)
 
 
-@pytest.fixture()
+@pytest.fixture(scope="class")
 def images(insert_data_from_csv):
     return insert_data_from_csv("./input/images.adb.csv")
+
+
+@pytest.fixture(scope="class")
+def retired_persons(db, insert_data_from_csv, utils):
```

Comment:
is this run for every test that uses "retired_persons" ? this will remove and re-insert tables every time, right? is that intended?

Reply:
in pytest there is a way to cache such results, with scope in fixture decorator. retired persons is run only once per class.

Reply:
cool!